src/script/media/MediaStreamErrorTypes.ts

```diff
@@ -29,3 +29,17 @@ export const MEDIA_STREAM_ERROR_TYPES = {
   ],
   PERMISSION: [MEDIA_STREAM_ERROR.NOT_ALLOWED_ERROR, MEDIA_STREAM_ERROR.SECURITY_ERROR],
 };
+
+export function isMediaStreamDeviceError(errorStr: string): boolean {
+  return MEDIA_STREAM_ERROR_TYPES.DEVICE.filter(errorType => errorType == errorStr).length > 0;
+}
+
+export function isMediaStreamReadDeviceError(errorStr: string): boolean {
+  return (
+    [
+      MEDIA_STREAM_ERROR.NOT_READABLE_ERROR,
+      MEDIA_STREAM_ERROR.NOT_ALLOWED_ERROR,
+      MEDIA_STREAM_ERROR.NOT_FOUND_ERROR,
+    ].filter(errorType => errorType == errorStr).length > 0
```

Comment:
I think this one will do the same result. So code will be more clean :)

```suggestion
    [
      MEDIA_STREAM_ERROR.NOT_READABLE_ERROR,
      MEDIA_STREAM_ERROR.NOT_ALLOWED_ERROR,
      MEDIA_STREAM_ERROR.NOT_FOUND_ERROR,
    ].includes(errorStr)
```